flecsi-sp/minimal/mesh.h

```diff
@@ -122,15 +161,21 @@ class minimal_mesh_t
   size_t
   indices(
     size_t index_space_id
+<<<<<<< HEAD
```

Comment:
I don't think the failures are because of this, but this _will_ cause a failure.